server/graphql/common/expenses.ts

```diff
@@ -1558,7 +1561,36 @@ const isValueChanging = (expense: Expense, expenseData: ExpenseData, key: string
   }
 };
 
-export async function editExpenseDraft(req: express.Request, expenseData: ExpenseData) {
+const isDifferentInvitedPayee = (expense: Expense, payee): boolean => {
+  const isInvitedPayee = !expense.data?.payee?.id && expense.data.payee.email;
+  if (isInvitedPayee) {
+    return !matches(expense.data.payee)(payee);
+  }
+  return false;
+};
+
+export async function sendDraftExpenseInvite(
+  req: express.Request,
+  expense: Expense,
+  collective: Collective,
+  draftKey: string,
+): Promise<void> {
+  const inviteUrl = `${config.host.website}/${collective.slug}/expenses/${expense.id}?key=${draftKey}`;
+  expense
```

Comment:
If we want the code that uses this function to be able to `await` on it

```suggestion
return expense
```